[2025-01-20T20:46:13.051+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-20T20:46:13.074+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DAG_AWS_PIPELINE.extract_events manual__2025-01-20T20:46:08.722287+00:00 [queued]>
[2025-01-20T20:46:13.084+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DAG_AWS_PIPELINE.extract_events manual__2025-01-20T20:46:08.722287+00:00 [queued]>
[2025-01-20T20:46:13.085+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-20T20:46:13.108+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_events> on 2025-01-20 20:46:08.722287+00:00
[2025-01-20T20:46:13.119+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=87) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-20T20:46:13.120+0000] {standard_task_runner.py:72} INFO - Started process 95 to run task
[2025-01-20T20:46:13.122+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'DAG_AWS_PIPELINE', 'extract_events', 'manual__2025-01-20T20:46:08.722287+00:00', '--job-id', '283', '--raw', '--subdir', 'DAGS_FOLDER/Ingesta_aws.py', '--cfg-path', '/tmp/tmpqaveqwb3']
[2025-01-20T20:46:13.123+0000] {standard_task_runner.py:105} INFO - Job 283: Subtask extract_events
[2025-01-20T20:46:13.185+0000] {task_command.py:467} INFO - Running <TaskInstance: DAG_AWS_PIPELINE.extract_events manual__2025-01-20T20:46:08.722287+00:00 [running]> on host 59ff16f1d4fb
[2025-01-20T20:46:13.320+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Wilfrido Rondon' AIRFLOW_CTX_DAG_ID='DAG_AWS_PIPELINE' AIRFLOW_CTX_TASK_ID='extract_events' AIRFLOW_CTX_EXECUTION_DATE='2025-01-20T20:46:08.722287+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-20T20:46:08.722287+00:00'
[2025-01-20T20:46:13.322+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-20T20:46:13.322+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-20T20:46:13.322+0000] {logging_mixin.py:190} INFO - Current task name:extract_events state:running start_date:2025-01-20 20:46:13.074633+00:00
[2025-01-20T20:46:13.323+0000] {logging_mixin.py:190} INFO - Dag name:DAG_AWS_PIPELINE and current dag run status:running
[2025-01-20T20:46:13.323+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-20T20:46:13.324+0000] {Ingesta_aws.py:119} INFO - Recuperando página 1...
[2025-01-20T20:46:13.791+0000] {Ingesta_aws.py:137} ERROR - Error al recuperar los datos. Código de estado: 404
[2025-01-20T20:46:13.792+0000] {Ingesta_aws.py:141} INFO - Transformando los datos a CSV...
[2025-01-20T20:46:13.799+0000] {Ingesta_aws.py:148} INFO - CSV guardado temporalmente en: /tmp/reports.csv
[2025-01-20T20:46:13.800+0000] {python.py:240} INFO - Done. Returned value was: /tmp/reports.csv
[2025-01-20T20:46:13.827+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-20T20:46:13.828+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=DAG_AWS_PIPELINE, task_id=extract_events, run_id=manual__2025-01-20T20:46:08.722287+00:00, execution_date=20250120T204608, start_date=20250120T204613, end_date=20250120T204613
[2025-01-20T20:46:13.848+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-20T20:46:13.848+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-20T20:46:13.849+0000] {logging_mixin.py:190} INFO - Dag name:DAG_AWS_PIPELINE queued_at:2025-01-20 20:46:08.735322+00:00
[2025-01-20T20:46:13.849+0000] {logging_mixin.py:190} INFO - Task hostname:59ff16f1d4fb operator:PythonOperator
[2025-01-20T20:46:13.904+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-20T20:46:13.931+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-20T20:46:13.935+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
